src/structures/MessageEmbed.js

```diff
@@ -7,11 +7,19 @@ const Util = require('../util/Util');
  * Represents an embed in a message (image/video preview, rich embed, etc.)
  */
 class MessageEmbed {
-  constructor(data = {}) {
-    this.setup(data);
+  /**
+   * @param {Object} data The data to build the embed from
+   * @param {boolean} skipValidation A flag to skip field validations
+   */
+  constructor(data = {}, skipValidation = false) {
+    this.setup(data, skipValidation);
   }
 
-  setup(data) {
+  /**
+   * @param {Object} data The data to build the embed from
+   * @param {boolean} skipValidation A flag to skip field validations
+   */
+  setup(data, skipValidation) {
```

Comment:
This method should either stay undocumented or get a docstring and a `@private`, since it's only for internal use.

Reply:
Will remove docstring completely as this isn't in the typings.